nservicebus/operations/performance-counters.md

```diff
@@ -111,6 +111,9 @@ Enabled by default and will only write to the counter if it exists.
 
 The NServiceBus Performance counters can be installed using the [NServiceBus PowerShell tools](management-using-powershell.md).
 
+## Perfomance Monitor Users local security group
+
+When [running installers](http://docs.particular.net/nservicebus/operations/installers) the service account will be automatically added to the local Performance Monitor Users group if executed with elevated privileges.
```

Comment:
links cant be fully qualified https://github.com/Particular/docs.particular.net#linking